configs/SIRMIXALOT/config.h

```diff
@@ -26,6 +26,16 @@
 #define BOARD_NAME        SIRMIXALOT
 #define MANUFACTURER_ID   CUST
 
+#define USE_ACC
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM20602
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+#define USE_BARO
+#define USE_BARO_BMP280
```

Comment:
The baro is on SPI3, not I2C.
```suggestion
#define USE_BARO_SPI_BMP280
```
![image](https://github.com/user-attachments/assets/fa64a059-7bb3-4188-bc06-a123f1181c01)

Reply:
Also need to add:
```
#define BARO_CS_PIN          PB9
```